zebra/rt_netlink.c

```diff
@@ -2281,7 +2281,7 @@ static struct nexthop netlink_nexthop_process_nh(struct rtattr **tb,
 		nh.ifindex = if_index;
 
 	*ifp = if_lookup_by_index_per_ns(zebra_ns_lookup(ns_id), nh.ifindex);
-	if (ifp)
+	if (*ifp)
```

Comment:
I see Donald already covered this here https://github.com/FRRouting/frr/pull/5240/commits/7134ba70601c5af70fa0342510f95521217028db

Reply:
Yes, he beat me to it. Will re-base and drop that commit once his goes in.